concepticondata/conceptlists/Hartmann-2013-162.tsv

```diff
@@ -1,163 +1,163 @@
+Hartmann-2013-162-88	88	JUMP	prygnutʹ	springen OR hüpfen	jump	1829	JUMP	The girl jumped.	A jumps	Core list	jump
+Hartmann-2013-162-89	89	JUMP OVER						The boy jumped over the fence.	A jumps over X	New meaning	jump-over
+Hartmann-2013-162-90	90	KILL	ubitʹ	umbringen OR töten	kill	1417	KILL	The man killed his enemy with a club.	A kills P (with I)	Core list	kill
+Hartmann-2013-162-91	91	KNOW	znatʹ	kennen	know	2248	KNOW (SOMEBODY)	The girl knew the boy.	A knows P	Core list	know
+Hartmann-2013-162-92	92	LAUGH	smejatʹsja	lachen	laugh	1355	LAUGH	The little girl laughed.	S laughs	Core list	laugh
+Hartmann-2013-162-93	93	LAUGH ABOUT						The woman laught about the dog.	A laughs about X	New meaning	laugh-about
+Hartmann-2013-162-94	94	LEARN				504	LEARN	The girl learnt the song.	A learns P	New meaning	learn
+Hartmann-2013-162-95	95	LEAVE	ujti OR uexatʹ	verlassen	leave	1752	LEAVE	The boy left the village.	A left L	Core list	leave
+Hartmann-2013-162-96	96	LIKE	nravitʹsja	mögen OR gefallen	like	1117	LIKE	The boy liked his new toy.	E likes M	Core list	like
+Hartmann-2013-162-97	97	LISTEN				1608	LISTEN	The boy listened to the music.	E listens to M.	New meaning	listen
+Hartmann-2013-162-98	98	LIVE	žitʹ	leben OR wohnen	live	363	DWELL (LIVE, RESIDE)	The old people live in town.	S lives somewhere (L)	Core list	live
+Hartmann-2013-162-99	99	LOAD	nagruzitʹ	laden	load	3669	LOAD (SOMETHING)	The farmer loaded hay onto the truck. = The farmer loaded the truck with hay.	A loads T (onto L)	Core list	load
+Hartmann-2013-162-100	100	LONG FOR				3473	LONG (FOR SOMETHING)	The teacher longs for coffee.	A longs for P	New meaning	long-for
+Hartmann-2013-162-101	101	LOOK AT	smotretʹ	anschauen	look	1819	LOOK	The boy looked at the girl.	A looks at P	Core list	look-at
+Hartmann-2013-162-102	102	MAKE		machen	make	734	MAKE	The man made a box out of wood.	A makes P (out of X)	Old meaning	make
+Hartmann-2013-162-103	103	MAKE APPEAR						The wizard made a white dove appear.	A makes P appear	New meaning	make-appear
+Hartmann-2013-162-104	104	MAKE CLIMB						The man made the monkey climb the wall.	C makes A climb L	New meaning	make-climb
+Hartmann-2013-162-105	105	MAKE HUNGRY						The hard work made the men hungry.	M makes E hungry	New meaning	make-hungry
+Hartmann-2013-162-106	106	MAKE ONE'S PRESENCE KNOWN						He made his presence known in the province.	S makes one's presence known	New meaning	make-ones-presence-known
+Hartmann-2013-162-107	107	MAKE RUN						The girl made the horse run.	A makes P run	New meaning	make-run
+Hartmann-2013-162-108	108	MAKE SICK						The cold made the girl sick.	M makes E sick	New meaning	make-sick
+Hartmann-2013-162-109	109	MARRY				203	MARRY	The man married the woman.	X marries Y	New meaning	marry
+Hartmann-2013-162-110	110	MEET	vstretitʹ	treffen	meet	1831	MEET	The men met the boys.	A meets X	Core list	meet
+Hartmann-2013-162-111	111	NAME	nazvatʹ	nennen	call OR name	1405	NAME	The parents called the baby Anna.	A name X (a) Y	Core list	name
```

Comment:
CALL BY NAME 180 ?

Reply:
Already done.

Reply:
Oh I see you mapped CALL BY NAME to Hartmann-2013-162-16. But wouldn't it be more appropriate to map it to Hartmann-2013-162-111 rather than to it's passive counterpart? Another reason for changing the mapping is that NAME 1405 is clearly a substantive.

Reply:
True, thanks, fixed.